massa-factory-worker/src/tests/tools.rs

```diff
@@ -1,67 +1,60 @@
+use massa_channel::sender::MassaSender;
+use massa_channel::MassaChannel;
 use massa_consensus_exports::MockConsensusController;
 use massa_models::config::MIP_STORE_STATS_BLOCK_CONSIDERED;
 use massa_versioning::versioning::MipStatsConfig;
 use massa_versioning::versioning::MipStore;
 use num::rational::Ratio;
 use parking_lot::RwLock;
 use std::sync::Arc;
+use std::thread::JoinHandle;
 
-use massa_factory_exports::{
-    test_exports::create_empty_block, FactoryChannels, FactoryConfig, FactoryManager,
-};
+use massa_factory_exports::{test_exports::create_empty_block, FactoryChannels, FactoryConfig};
 use massa_models::{address::Address, block_id::BlockId, prehash::PreHashMap, slot::Slot};
 use massa_pool_exports::MockPoolController;
 use massa_pos_exports::MockSelectorController;
 use massa_protocol_exports::MockProtocolController;
 use massa_signature::KeyPair;
 use massa_storage::Storage;
-use massa_time::MassaTime;
 
-use crate::start_factory;
+use crate::block_factory::BlockFactoryWorker;
+use crate::endorsement_factory::EndorsementFactoryWorker;
 use massa_wallet::test_exports::create_test_wallet;
 
 /// This structure store all information and links to creates tests for the factory.
-/// The factory will ask that to the the pool, consensus and factory and then will send the block to the consensus.
-/// You can use the method `new` to build all the mocks and make the connections
 #[allow(dead_code)]
-pub struct TestFactory {
```

Comment:
Is it not possible to remove the dead_code with a feature/cfg?

Reply:
(same for the endorsment test factory)

Reply:
Done